includes/Plugin.php

```diff
@@ -200,6 +200,7 @@ function() use ( $options, $activation_flag ) {
 				( new Core\Feature_Tours\Feature_Tours( $this->context, $user_options ) )->register();
 				( new Core\Util\Migration_1_3_0( $this->context, $options, $user_options ) )->register();
 				( new Core\Util\Migration_1_8_1( $this->context, $options, $user_options, $authentication ) )->register();
+				( new Core\Dashboard_Sharing\Activity_Metrics( $this->context, $user_options ) )->register();
```

Comment:
This hasn't been asked in AC, but I think we should introduce the core class for the dashboard sharing functionality (similar to the Authentication stuff) and define dashboard sharing parts. So, let's create `Core/Dashboard_Sharing/Dashboard_Sharing` class that registers everything else and then register the base class in the Plugin.